src/components/LeftSidebar/NewGroupConversation/Confirmation/Confirmation.vue

```diff
@@ -83,42 +85,11 @@ export default {
 			type: Boolean,
 			required: true,
 		},
-		linkToConversation: {
-			type: String,
-			required: true,
-		},
-	},
-
-	data() {
-		return {
-			showTooltip: false,
-			confirmationText: '',
-			showConfirmationText: false,
-		}
 	},
 
 	methods: {
-		async onClickCopyLink() {
-			try {
-				await navigator.clipboard.writeText(this.linkToConversation)
-				this.onCopy()
-			} catch (error) {
-				this.onError()
-			}
-		},
-		onCopy() {
-			this.confirmationText = t('spreed', 'Link copied to the clipboard!')
-			this.showConfirmationText = true
-			setTimeout(function() {
-				this.showConfirmationText = false
-			}, 800)
-		},
-		onError() {
-			this.confirmationText = t('spreed', 'Error')
-			this.showConfirmationText = true
-			setTimeout(function() {
-				this.showConfirmationText = false
-			}, 800)
+		onClickCopyLink() {
```

Comment:
Shouldn't this method be asymc as well as the others?

Reply:
`async/await` is not required here because we never need to wait copy.

For consistency I removed `await` everywhere for this method. I preferred removing await everywhere than adding here because:
- As we transpile to old ES5, async/await adds really much code after the transpiling
- We always can do copying "in parallel" while await attracts sequential execution